### PR TITLE
Allow generic inference for ASR for superb

### PR DESCRIPTION
**Pull request by osanseviero**
The API specification is in `model.py` and people specify additional requirements with a text file.

Example repos
-  https://huggingface.co/osanseviero/asr-with-transformers-wav2vec2
-  https://huggingface.co/osanseviero/asr-with-s3prl
-  https://huggingface.co/osanseviero/asr-with-s3prl_req

### CODE REVIEW

File: api-inference-community/tests/test_dockers.py [hunk 1]
```diff
@@ -211,6 +221,7 @@ def framework_docker_test(self, framework: str, task: str, model_id: str):
             self.assertEqual(response.content, b'{"ok":"ok"}')
 
             response = httpx.post(url, data=b"This is a test", timeout=timeout)
+            print(response.json())
```
Comment (Narsil) [inline api-inference-community/tests/test_dockers.py:224]: Oops :)

File: api-inference-community/tests/test_dockers.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 class DockerPopen(subprocess.Popen):
     def __exit__(self, exc_type, exc_val, traceback):
         self.terminate()
-        self.wait(5)
+        self.wait(30)
```
Comment (Narsil) [inline api-inference-community/tests/test_dockers.py:15]: NIT: Here it's 30, later it's 20, can we keep consistent ?

**Review by Narsil — APPROVED** [review]
LGTM.

1 NIT

**Review by lewtun — APPROVED** [review]
LGTM 🚀 !

**Comment by osanseviero** [comment]
Unit tests pass now, including docker test that does not use `s3prl`. For the `s3prl` test, the container gets `Killed` after downloading the upstream model from the Torch Hub. 

@Narsil or @lewtun, could you try this command if possible to see if it's reproducible? I've find my computer to have a couple of issues with Docker. Example to reproduce
```
./manage.py docker --framework superb --task automatic-speech-recognition --model-id osanseviero/hubert_s3prl_req
```

**Pull request merged**